### DISCUSSION TITLE
C'est quoi Angular ? | Bonjour Angular

### DISCUSSION THREAD

**Discussion by giscus** [bonjour-angular/apprendre#4] [General]
# C'est quoi Angular ? | Bonjour Angular

C'est quoi Angular ?

https://apprendre.bonjour-angular.com/cest-quoi/angular/

**Comment by hsawadogo08** [comment]
Quelles sont les avantages de Angular par rapport aux autres Frameworks frontend ?

**Reply by misterprog-dev** [reply]
@hsawadogoh, je crois que l'un des plus gros avantage d'Angular est sa structure interne. Il est plus structuré et encourage les bonnes pratiques. En termes de sécurité applicative, on a moins de vulnérabilité vu qu'il y a des mises à jour régulières.

**Comment by murhulayves** [comment]
Bonjour 👋 
J'aimerais apprendre beaucoup plus sur Angular
